### ISSUE TITLE
Lucid dream minor rank 1 gets simmed as minor rank 3

### ISSUE THREAD

**Issue by REVERSE-MISAYA** [seriallos/raidbots-issues#168]
**Describe the bug**
when simming a character with "memory of lucid dreams (rank 1) minor essence" it gets simmed as if the player had rank 3 minor essence. Thus resulting in an incorrect sim, because the character got an extra buff which he was not supposed to have. (tested it on a tauren boomkin)

**To Reproduce**
1. sim with lucid dreams minor rank 1
2. after sim check buff uptime tab, look for "Lucid Dreams" buff
3. you will see player got x-times rank 3 buff while only having rank 1

**Expected behavior**
not receiving lucid dreams buff if the minor essence rank is lower than 3

**Link a sim with the problem (if available)**
https://www.raidbots.com/simbot/report/fCZDnpbbb3hLW7TWt98DVa
[](https://www.raidbots.com/simbot/report/fCZDnpbbb3hLW7TWt98DVa)

**Comment by seriallos** [comment]
Pretty sure this a display bug and that the rank 3 isn't actually being simmed.

Looking at the Full HTML Report for the sim you linked, the Lucid Dreams buff isn't shown: https://www.raidbots.com/simbot/report/2Fc63DgYeSCq2eQVz2S81Y/simc

Looking at that buffs section, the buff isn't there:
![image](https://user-images.githubusercontent.com/1010294/73603579-68af6900-4539-11ea-8fc6-8ad1b0c772ca.png)


I ran another sim where I manually increased your Lucid Dreams to rank 3 and it's both higher DPS and has the buff with the details about the vers:

https://www.raidbots.com/simbot/report/78hk8G7D8cCpnm9ygwAnW3/simc

![image](https://user-images.githubusercontent.com/1010294/73603581-71a03a80-4539-11ea-979c-f7aee3fadccc.png)

It looks like SimC has a really subtle difference in how buffs are exported to the HTML vs the JSON. My bet is that the buff you see on the Raidbots report is basically a "null" buff that doesn't actually do anything in the sim and that the HTML report properly filters it out.
